scala_modules/build.sbt

```diff
@@ -19,24 +18,22 @@ lazy val awsVersion = "1.7.4"
 
 lazy val events = project
   .settings(
-      name := "events",
-      assemblyMergeStrategy in assembly := {
-        case x if x.endsWith("io.netty.versions.properties") => MergeStrategy.first
-        case x =>
-          val oldStrategy = (assemblyMergeStrategy in assembly).value
-          oldStrategy(x)
-      },
-      resolvers += Resolver.sonatypeRepo("releases"),
-      addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
```

Comment:
dropping compiler plugin was intentional, this is unused - didn't end up using macros